### DISCUSSION TITLE
Error: network size may only be between /16 and /31

### DISCUSSION THREAD

**Discussion by a77z88** [community/community#78468] [Programming Help]

**Comment by davea42** [comment]
I'll make a guess as to what you are asking, since you provide no context.

When defining an IPV4 network interface one will specify 3 things, in general.

One of those things is a netmask.

The usual is 255.255.255.0 which means /24, so just 8 bits of network id
(meaning 256 network ids) apply
to the network you want available. For most people this is all you have and all you
can possibly use.

255.255.0.0 is /16 and means 16 bits of network id are in the specified network.
That is quite a large network.

It makes no practical sense to ask for a range outside of this in any sensible normal
context.  Perhaps take a look at wikipedia.org/wiki/Subnet ,

I hope this is somehow relevant.
